translations/en.json

```diff
@@ -547,6 +548,7 @@
   "service.metadata.spatialExtent": "Spatial extent",
   "share.tab.permalink": "Share",
   "share.tab.webComponent": "Integrate",
+  "show.all.text": "Show all text",
```

Comment:
these keys are not very clear, how about:

* `ui.showAllText`
* `ui.hideFullText`

This is consistent with `ui.readMore` and `ui.readLess`